### DISCUSSION TITLE
Where are the docs for plugins?

### DISCUSSION THREAD

**Discussion by cottrell** [perspective-dev/perspective#2344] [Q&A] [answered]
For example, where to find the full list of options on a d3_x_bar plugin for example. Trying to use this from Panel if that makes sense. Everything might have been renamed. Feels like the code I'm reading does not contain the API, so maybe this actually points to an external implementation?

**Comment by cottrell** [comment]
Ok, maybe it's all here (for d3fc) and it's just a matter of understand what and how options are passed through

https://github.com/finos/perspective/tree/master/packages/perspective-viewer-d3fc/src/js/charts

**Comment by cottrell** [comment]
And here is a good place to start. Probably the docs should point to this 

https://github.com/finos/perspective/blob/master/docs/src/components/Demo/layouts.js

It is not obvious what "group_by" is immediately.

**Reply by timkpaine** [reply, marked as answer]
? https://perspective.finos.org/docs/view/#group-by

**Reply by cottrell** [reply]
yes but in the case I was looking at you need to use group by simply to make the index appear without any aggregation so it wasn't obvious.
